File: Ejercicios Propuestos 3/Ejercicio 10/ejercicio10.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cout << "Ingrese la cantidad de elementos: ";
    cin >> n;

    int* v = new int[n];

    for (int i = 0; i < n; i++) {
        cout << "Elemento " << i << ": ";
        cin >> v[i];
    }

    cout << "Vector ingresado: ";
    for (int i = 0; i < n; i++) {
        cout << v[i] << " ";
    }
    cout << endl;

    delete[] v;
    return 0;
}

File: Ejercicios Propuestos 3/Ejercicio 10/Ejercicio 10.jl
cantidad = 0
i = 0

println("Ingrese la cantidad de elementos del vector:")
cantidad = parse(Int, readline())

# Crear vector dinámico
puntero = Vector{Int}(undef, cantidad)

println("Ingrese los elementos:")
for i = 1:cantidad
    println("Elemento ", i - 1, ":")
    puntero[i] = parse(Int, readline())
end

println("Elementos del vector dinámico:")
for i = 1:cantidad
    print(puntero[i], " ")
end
println()

# Liberar el vector
puntero = nothing
